backend/src/api/InviteController.ts

```diff
@@ -70,6 +70,13 @@ export default class InviteController {
                 return response.error('invalid-code', 'Invite code not found or already used');
             }
 
+            const issuerRestrictions = await this.userManager.getUserRestrictions(invite.issued_by);
+            if (!issuerRestrictions.canInvite) {
+                this.logger.warn(`Invite issuer ${invite.issued_by} has no permission to invite`,
+                    {invite: code});
+                return response.error('invalid-code', 'Invite code can\'t be used');
```

Comment:
I started addressing it and realized that for consistency all similar constant codes should be extracted as well, but it will introduce a bit of a scope creep. 

So I decided to create an issue so we don't forget about this: #105